index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Laura's site</title>
+
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/index_style.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html"> Home </a></li>
+        <li><a href="portfolio.html"> Portfolio </a></li>
+        <li><a href="journal.html"> Journal </a></li>
+        <li><a href="About.html"> About </a></li>
+      </ul>
+    </nav>
+    <h1> Laura's Site</h1>
+
+  </header>
+
+  <main>
+    <section>
+      <div class="left">
+        <h2> Laura Melgarejo Silva </h2>
+        <p> Colombian. physicist. Adventurer. Aspiring coder.
+        </p>
+        <h4><a href="About.html">Find more</a></h4>
+      </div>
+      <img src="assets/pics/laura_salchipapa.jpg" alt="laura_photo">
+    </section>
+    <section>
+
+    </section>
+  </main>
+
+  <footer>
+    <nav>
+      <span>Connect:</span>
+      <ul>
+        <li><a href="https://www.facebook.com/" target="_blank"><img src="assets/icons/1489803528_icon_facebook_24.ico" alt="facebook"></a></li>
+        <li><a href="https://www.tweeter.com/" target="_blank"> <img src="assets/icons/1489803548_icon_twitter_24.ico" alt="tweeter"> </a></li>
+        <li><a href="https://www.instagram.com/" target="_blank"><img src="assets/icons/1489803534_icon_instagram_24.ico" alt="instagram"></a></li>
+      </ul>
+    </nav>
+    <h6> Copyright &copy; 2017-Laura Melgarejo Silva</h6>
+  </footer>
+
```

Comment:
I like the use of whitespace here to separate the footer section from the rest.  I think it is good practice to have whitespace between different sections of code.  :)